### DISCUSSION TITLE
Question about recent fix for turning off repeating looping notes

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1213] [Team Posts]
Hi. Recently there was a quick fix I recall that turned off repeating looping notes
 or something like that.
I remember it being discussed (at one point you were worried it might interfere
 with my midi looping fixes) but I can't find any mention of it now, here or in the
 issues or the ChangeLog.
Can you guys recall this and point me to the commit so I can have a look?
Thanks.

**Comment by spamatica** [comment]
Guessing "you" in this context is @kybos because I have no idea :P

**Comment by kybos** [comment]
I think what you mean is this (it's not really about looping, rather suppressing playback in replace mode in general):
https://github.com/muse-sequencer/muse/issues/836
https://github.com/muse-sequencer/muse/issues/789

The commit:
https://github.com/muse-sequencer/muse/commit/bbf2741fc9a47a832a8cf1c073b6dc39ae58bf26

From the changelog:
12.02.2021
...
    - Fix #836: Don't play existing midi events in replace mode. (kybos)
